Portfolio/src/Wrapper.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+</script>
+
+<template>
+    <div class="wrapper">
+
+        <slot></slot>
```

Comment:
The indentation of your markup is a bit off. In general it is important for developers to have a consistent and readable markup style, that's why we use the prettier extension on VScode for example. So work a bit more on the spaces between lines of code and prettify the code a bit :). THen it will be easier for u to write better code

Reply:
Done